tests/publiccloud/run_ltp.pm

```diff
@@ -141,19 +156,47 @@ sub run {
 
     assert_script_run($log_start_cmd);
 
-    my $cmd = 'perl -I runltp-ng runltp-ng/runltp-ng ';
-    $cmd .= '--logname=ltp_log --verbose ';
-    $cmd .= '--timeout=1200 ';
-    $cmd .= '--run ' . get_required_var('LTP_COMMAND_FILE') . ' ';
-    $cmd .= '--exclude \'' . get_var('LTP_COMMAND_EXCLUDE') . '\' ' if get_var('LTP_COMMAND_EXCLUDE');
-    $cmd .= '--backend=ssh';
-    $cmd .= ':user=' . $instance->username;
-    $cmd .= ':key_file=' . $instance->ssh_key;
-    $cmd .= ':host=' . $instance->public_ip;
-    $cmd .= ':reset_command=\'' . $reset_cmd . '\'';
-    $cmd .= ':ssh_opts=\'-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no\' ';
-    $cmd .= '--json_filter=openqa ';
-    assert_script_run($cmd, timeout => get_var('LTP_TIMEOUT', 30 * 60));
+    my $ltp_timeout = get_var('LTP_TIMEOUT', 30 * 60);
+    my $ltp_suite_timeout = 1200;
```

Comment:
first of all as it was clarified with @acerv  this is not "suite timeout" but "test timeout" . and also since this variable used in single place we can just do `$cmd .= '--timeout=1200 ';` ( as it was before ) and no need to introduce variable for this

Reply:
ok